niri-config/src/lib.rs

```diff
@@ -3539,6 +3541,8 @@ mod tests {
 
                 tablet {
                     map-to-output "eDP-1"
+                    calibration-matrix 1.0 0.0 0.0 \
+                                       0.0 1.0 0.0
```

Comment:
Do you mind changing this to something like 1 2 3 4 5 6 so we verify that we get the values in expected places.